src/main/java/uk/ac/ic/wlgitbridge/snapshot/push/exception/PostbackTimeoutException.java

```diff
@@ -13,7 +13,7 @@
 
     @Override
     public String getMessage() {
-        return "timeout (60s)";
+        return "timeout";
```

Comment:
This gets printed to the client, perhaps it would be better as "Timed out"? Is there a standard phrasing for that?

Reply:
I like the idea of making it more friendly. Maybe `Request Timed Out (after #{TIMEOUT}s)`?